@kiva/kv-components/vue/KvContentfulImg.vue

```diff
@@ -213,17 +213,32 @@ export default {
 
 		const buildUrl = (image = null, multiplier = 1) => {
 			let src = image && image.url ? `${image.url}?` : `${contentfulSrc.value}?`;
-			const imgWidth = image ? image.width : width.value;
-			const imgHeight = image ? image.height : height.value;
+			let imgWidth = (image ? image.width : width.value) * multiplier;
+			let imgHeight = (image ? image.height : height.value) * multiplier;
+
+			// The max contentful image size is 4000px so we have to
+			// impose a limit of 2000px here for both height and width
+			// so when we request the retina x2 image we don't go over the 4000px limit
+			let newMultiplier;
+			if (imgWidth >= 2000) {
+				newMultiplier = imgWidth / 2000;
+				imgWidth = 1999;
```

Comment:
Should `2000` and `1999` match here to ensure we aren't potentially off by 1px?

Reply:
Party like its 1999

Reply:
Haha.

Did something change with the last force push? Or does it not matter as much that the multiplier doesn't use 1999?

Reply:
Sorry, fixed